idea/vara-eth/frontend/src/features/codes/lib/queries.ts

```diff
@@ -1,56 +1,51 @@
 import { useQuery } from '@tanstack/react-query';
-import { request } from 'graphql-request';
 
 import { EXPLORER_URL } from '@/shared/config';
-import { graphql } from '@/shared/graphql';
+import { fetchWithGuard } from '@/shared/helpers';
 
 export const CODE_STATUS = {
-  VALIDATION_REQUESTED: 'validation_requested',
-  VALIDATION_FAILED: 'validation_failed',
-  VALIDATED: 'validated',
+  VALIDATION_REQUESTED: 'ValidationRequested',
+  VALIDATION_FAILED: 'ValidationFailed',
+  VALIDATED: 'Validated',
 } as const;
 
-const GET_CODES_QUERY = graphql(`
-  query GetCodes($first: Int!, $offset: Int!) {
-    allCodes(first: $first, offset: $offset) {
-      nodes {
-        id
-        status
-      }
-      totalCount
-    }
-  }
-`);
+export type Code = {
+  id: string;
+  status: string;
+  createdAt: string;
+};
+
+type CodesResponse = {
+  data: Code[];
+  total: number;
+  limit: number;
+  offset: number;
```

Comment:
I guess it can be a separate type like `ListResponse`. And types like this could extend it